guides/common/attributes.adoc

```diff
@@ -10,9 +10,6 @@
 // The above attribute should point to the GA version number (x.y) for all releases including Beta
 :SatelliteAnsibleVersion: 2.9
```

Comment:
This one did jump out to me. Looks like it's unused since eb0b825d2a29c3bf0d195f8a854da375037073be. Is it still used in some downstream doc or can we get rid of it? Doesn't have to be in this PR, but can you notice I'm on a quest to reduce the number of attributes? :)

Reply:
It is not used on master but it is used on 3.2 and 3.1. I suggest this to be removed in a separate PR against master and cherry-picked to 3.{5,4,3}